src/extensions/markdown/Link/plugins/LinkTooltipPlugin/index.tsx

```diff
@@ -239,6 +239,8 @@ class SelectionTooltip implements PluginView {
                 view.state.tr.addMark(from, to, linkType(view.state.schema).create({href: url})),
             );
 
+            view.dispatch(view.state.tr.setSelection(TextSelection.create(view.state.doc, to)));
```

Comment:
1. use this code
```
        if (normalizeResult) {
            const {url} = normalizeResult;
            const {from, to} = textNode;

            const tr = view.state.tr;
            tr.setSelection(TextSelection.create(tr.doc, tr.mapping.map(to)));
            tr.addMark(from, to, linkType(view.state.schema).create({href: url}));
            view.dispatch(tr);
        }
```
2. you no need `setTimeout` more
3. this logic in update, removePlaceholderLink